packaging/suse/container/Dockerfile

```
@@ -28,7 +28,7 @@ ENV VERSION=%%VERSION%%
 RUN mix phx.digest
 RUN mix release
 
-FROM bci/bci-base:15.4 AS trento
+FROM bci/bci-base:15.6 AS trento
```

same as above (also no need for step alias since it's the last step)
```suggestion
FROM registry.suse.com/bci/bci-base:15.6
```